/home/jdamr/agv_ws/src/Project-AGV-main/stage2_slam/cartographer_agv/urdf/agv.urdf
jdamr@jdamr-pc:~/agv_ws$ ros2 launch cartographer_agv agv_launch.py
[INFO] [launch]: All log files can be found below /home/jdamr/.ros/log/2025-04-02-10-16-59-085099-jdamr-pc-8115
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught multiple exceptions when trying to load file of format [py]:
 - FileNotFoundError: [Errno 2] No such file or directory: '/home/jdamr/agv_ws/src/cartographer_agv/share/cartographer_agv/urdf/agv.urdf'
 - InvalidFrontendLaunchFileError: The launch file may have a syntax error, or its format is unknown
